src/frontend/src/service/api-service.ts

```diff
@@ -11,6 +11,7 @@ export type Request = {
   method: Method;
   headers?: object;
   authenticated?: boolean;
+  withCredentials?: boolean;
```

Comment:
What does `withCredentials` do and why do we need it?

Reply:
`withCredentials` is the way you are able to make a request that passes cookies in axios

https://stackoverflow.com/a/43178070

[Axios github docs:](https://github.com/axios/axios)
```
  // `withCredentials` indicates whether or not cross-site Access-Control requests
  // should be made using credentials
  withCredentials: false, // default
```